src/main/java/school/lemon/changerequest/java/vehicles/Airplane.java

```diff
@@ -0,0 +1,85 @@
+package school.lemon.changerequest.java.vehicles;
+
+public class Airplane extends Default implements IAirplane {
+
+    private int maximumHeightFeet;
+
+    Airplane(int manufacturedYear, String make, String model, int maximumHeightFeet) {
+        super(manufacturedYear, make, model);
+        this.maximumHeightFeet = maximumHeightFeet;
+    }
+
+    @Override
+    public int getManufacturedYear() {
```

Comment:
Please move all getters/setters for manufacturedYear, make, model to parent class. Do it for every child.